### ISSUE TITLE
tuplet@slope

### ISSUE THREAD

**Issue by craigsapp** [music-encoding/music-encoding#1195]
It would be useful to add `@slope` to `<tuplet>` and `<tupletSpan>` to allow for explicit horizontal tuplet brackets.  This would be similar to the existing `beam@slope` and `beamSpan@slope`.

See https://github.com/rism-digital/verovio/pull/3442 for discussion.

**Comment by pe-ro** [comment]
@craigsapp, is there a need for separate `@slope` and `@bracket.slope` attributes?  This would make it possible to encode separate beam slope and bracket slope, if the following example had sloped beams but flat brackets
![image](https://github.com/music-encoding/music-encoding/assets/5712877/75463a0f-e82b-4be8-9c0a-7f7eb30db0f8)
Ideally, such things wouldn't occur, but I can imagine there's "incorrect" notation in the world that has such a feature.

**Comment by pe-ro** [comment]
Doh!  Never mind that last comment.  Beams and brackets are separate already.  I was led astray by the presence of `@bracket.place` and `@bracket.visible` in `att.tuplet.vis`.  Mea culpa.

**Comment by pe-ro** [comment]
How about adding a `@bracket.slope` attribute on `<tuplet>` and `<tupletSpan>` in their respective `*.vis` classes?  Using this name will make it clear that the attribute controls the slope of the bracket only, not the tuplet beam.  The slope of the tuplet beam will continue to be indicated by `@slope` on the `<beam>` element within `<tuplet>`.

**Comment by craigsapp** [comment]
Yes, that is good.  Most brackets will be expected to be horizontal, but `bracketSpan@slope` can be used to make it non-horizontal.
